capitalize_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
```

Comment:
- [ ] Kindly remember to `require_relative './decorator'` so that the Decorator file can be accessed.